The-Chonky-Panda/Atelier
# super minor style update

## Description

super minor style update to make everything center a little bit.

## How to test

@shabbyblue16  @surekhaw Could you both check whether this breaks how you want your widget to look like? I'm attempting to make everything look more align, but I'm not as familiar with the specific css for your widgets as you are. Everything looks fine on my external monitor which is larger, but the style list still looks a little bit packed on my laptop. I don't want them to look bad on your screen when you are presenting.

## Notes

If it doesn't work out I can easily revert the changes. So no pressure!

## Issue tracking
